README.md

```diff
@@ -9,7 +9,7 @@ Read all about it at http://pitest.org
 
 ### 1.3.3-SNAPSHOT
```

Comment:
You may wish to name it 1.4.0 instead of 1.3.3

Reply:
Yes, that was/is the plan

Reply:
now fixed